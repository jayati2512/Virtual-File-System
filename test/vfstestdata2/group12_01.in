createvfs ROOT 1024
mountvfs ROOT
makedir / home
makedir /home/ a
makedir /home/a/ b
makedir /home/ c
makedir /home/ d
addfile /home/a/b/ add1.txt /usr/include/stdio.h
addfile /home/a/b/ add2.txt /usr/include/stdio.h
listfile /home/a/add1.txt /tmp/out1.txt
listfile /home/a/b/add2.txt /tmp/out2.txt
makedir / d
deletedir /home/c/
deletedir /c/
addfile /home/a/b/ addout2.txt /usr/include/stdio.h
removefile /home/a/b/addout1.txt
makedir /home/ z
exportfile /home/a/b/addout.txt expout.txt
listfile /home/a/b/addout.txt /usr/include/listfileout.txt
movefile /home/a/b/addout2.txt /home/d
