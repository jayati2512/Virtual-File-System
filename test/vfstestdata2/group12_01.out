createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
listdir_FAILURE SOURCE_FILE_PATH_NOT_FOUND
listfile_SUCCESS
makedir_SUCCESS
deletedir_SUCCESS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_DIR
addfile_SUCCESS
removefile_SUCCESS
makedir_SUCCESS
exportfile_SUCCESS
listfile_SUCCESS
movefile_SUCCESS
